src/test/java/org/folio/services/migration/item/ItemShelvingOrderMigrationServiceTest.java

```diff
@@ -30,7 +31,7 @@ public void shouldTriggerMigration(String from, String to) {
   }
 
   @Parameters({
-    "20.2.0, 20.2.1",
+    "20.2.1, 20.2.2",
```

Comment:
Is this because the initial enablement now starts at 20.2.1?

(and does this mean that these tests will need to change every time a new version is released?)

Reply:
> Is this because the initial enablement now starts at 20.2.1?

Yes

> (and does this mean that these tests will need to change every time a new version is released?)

No, unless we change from module version for the script.

Reply:
> No, unless we change from module version for the script.

Ok, thanks for confirming that